style.css

```diff
@@ -250,3 +236,329 @@ body {
   display: inline-block;
   width: 20px;
 }
+
+/* Recent Works section */
+
+.recent-work {
+  background-color: white-background-color;
+  padding: 80px 0;
+  position: relative;
+}
+
+.about-me {
+  position: relative;
+  top: 50px;
+}
+
+.recent-work-container {
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+}
+
+.section-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.recent-work h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  align-items: center;
+  margin-bottom: 40px;
+  color: headline-color;
+}
+
+.title-container {
+  position: relative;
+}
+
+.section-indicator {
+  display: inline-block;
+  width: 48px;
+  height: 8px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  bottom: -3px;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 30px;
+}
+
+.card-grid {
+  display: grid;
+  grid-template-areas:
+    "card-img card-img card-img"
+    "card-title card-title card-title"
+    "tags tags tags"
```

Comment:
- [ ] I would suggest you make your design as close to the original design template as much as possible. The cards should have the margins (top, bottom, left, right) defined in the design and also font and colours (background and font) must be the same as defined on the template.

> Template design
 
<img width="256" alt="Screenshot 2023-04-29 at 13 25 41" src="https://user-images.githubusercontent.com/71140133/235302625-29856733-c228-462e-b878-a322904934f8.png">

> Your design
<img width="375" alt="Screenshot 2023-04-29 at 13 25 15" src="https://user-images.githubusercontent.com/71140133/235302663-f3c66e2e-5edd-41eb-bfc9-660c7f8934e3.png">

- [ ] The `see project ` buttons should be the same size, color, and font as indicated in the original design